tests/phpunit/elementor/includes/base/test-widget-base.php

```diff
@@ -0,0 +1,56 @@
+<?php
+namespace Elementor\Tests\Phpunit\Includes\Base;
+
+use Elementor\Plugin;
+use Elementor\Testing\Elementor_Test_Base;
+use Elementor\Core\Frontend\Render_Mode_Manager;
+use Elementor\Tests\Phpunit\Includes\Base\Mock\Mock_Widget;
+use Elementor\Tests\Phpunit\Includes\Base\Mock\Mock_Static_Render_Mode;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Test_Widget_Base extends Elementor_Test_Base {
+	public function test_render_content_should_render_static() {
+		require_once __DIR__ . '/mock/mock-widget.php';
```

Comment:
Autoload?

Reply:
Not working in tests. 
The core files are loaded throw the autoloader.
The tests file loaded only when there is a 'test-' prefix